frontend/src/modules/products/components/ProductForm.tsx

```diff
@@ -30,8 +32,21 @@ export const ProductForm: React.FC<ProductFormProps> = ({
   const initialValues: CreateProductDTO = {
     name: '',
     price: 0,
-    bulk: false,
-    bulkQuantity: 0,
+    bulk: true,
+    bulkQuantity: undefined,
+  };
+  const getUser = useGlobalStore((state) => state.getUser);
+  const companyType = getUser()?.company?.type;
+
+  const isSuppManu = () => {
+    if (
+      companyType === CompanyDTOTypeEnum.Supplier ||
+      CompanyDTOTypeEnum.Manufacturer
+    ) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
also, since we reuse this logic, we can extract it to its own function. preferably, if you could extract this to its own custom hook, that would be great :). alternatively, you can also just extract it to a utility function and pass in the companyType as a parameter.